packages/frontend/core/src/modules/workbench/view/desktop-adapter.ts

```diff
@@ -36,8 +37,19 @@ export function useBindWorkbenchToDesktopRouter(
     ) {
       return;
     }
-    workbench.open(newLocation);
+    // skipping default location initialization since we will init the views
```

Comment:
This will make it impossible to open the page specified in the workbench through the URL.

Reply:
need more thoughts on the solution